src/test/groovy/io/vertx/ext/hawkular/impl/NetClientITest.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2015 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.ext.hawkular.impl
+
+import io.vertx.groovy.core.net.NetClient
+import io.vertx.groovy.ext.unit.TestContext
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+import java.util.concurrent.ForkJoinPool
+
+/**
+ * @author Thomas Segismont
+ */
+class NetClientITest extends BaseITest {
+  static final RESPONSE_CONTENT = 'some text'
+
+  def testHost = '127.0.0.1'
+  def testPort = getPort(9194)
+  def metricPrefix = "${METRIC_PREFIX}.vertx.net.client.${testHost}:${testPort}."
+
+  def concurrentClients = ForkJoinPool.commonPool().parallelism
+  def List<NetClient> netClients = []
+
+  @Before
+  void setup(TestContext context) {
+    def verticleName = 'verticles/net_server.groovy'
+    def instances = 1
+    def config = [
+      'host'   : testHost,
+      'port'   : testPort,
+      'content': RESPONSE_CONTENT
+    ]
+    deployVerticle(verticleName, config, instances, context)
```

Comment:
shouldn't you wait that the verticle instances are deployed before starting the test ?